packages/core/src/components/button/button.scss

```diff
@@ -53,14 +53,10 @@
     &.-action--secondary {
       border-color: helpers.color('border-action');
 
-      /* stylelint-disable max-nesting-depth */
-      @media (hover: hover) {
-        &:hover,
-        &:active {
-          border-color: helpers.color('border-action-hover');
-        }
+      &:hover,
```

Comment:
We still want to wrap `:hover` with the media query though, right?
So maybe we need two rules now

```scss
@media (hover:hover) {
    &:hover {
      border-color: helpers.color('border-action-hover');
    }
}

&:active {
  border-color: helpers.color('border-action-hover');
}
```